lib/uri/common.rb

```diff
@@ -68,16 +68,30 @@ module Schemes
   end
   private_constant :Schemes
 
+  # Registers the given +klass+ as the class to be instantiated
+  # when parsing a \URI with the given +scheme+:
```

Comment:
It might be a good idea to keep the information about what kinds of strings are allowed as `scheme`.

Reply:
Restored.